tdrs-backend/manifest.buildpack.yml

```diff
@@ -0,0 +1,9 @@
+version: 1
+applications:
+- name: tdp-backend
+  memory: 256M
+  instances: 1
+  disk_quota: 2G
```

Comment:
since the [dev env ADR was approved](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/Architecture%20Decision%20Record/015-development-environments.md), can this be safely reduced to 1GB?